kvlds/btree_node.c

```diff
@@ -44,6 +44,8 @@ struct descend {
 
 static int callback_fetch(void *, int, int, const uint8_t *);
 static int callback_descend(void *);
+static int btree_node_fetch_canfail(struct btree * T, struct node * N,
+    int (* callback)(void *), void * cookie, int canfail);
```

Comment:
Don't need a forward declaration since this the function is only used after it's defined.